CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: selecting ``os``, ``network``, ``fs`` or ``process`` from ``sys.nodes``
+   in very quick succession or multiple times in same select statement
+   could lead to wrong results for percentual values from sigar
```

Comment:
suggest: "from the underlying sigar library." (also missing . at the end of the sentence)